### PR TITLE
Test from loadtest box

### PR DESCRIPTION
**Pull request by maxkadel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,3 +31,68 @@ There are capistrano tasks to connect to the Solr UI for managing solr that can
...
+```bash
+jmeter -n -t solr_test_plan.jmx
+```
+* Copy the results to your local machine (from the local)
+```bash
+scp deploy@loadtest.lib.princeton.edu:~/test_results/simple_data_writer.csv ./test_results
+```
+## User Defined Variables by Environment
+### Development
+host orangelight.dev.solr.lndo.site
```
Comment (ishasinha1) [inline README.md:63]: The formatting on this is running everything together. Should it be a list?

File: keywords.csv [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+pier
+collect
+recommendation
+emotion
+disagree
+twist
+photocopy
+load
+blow
+lick
```
Comment (ishasinha1) [inline keywords.csv]: Noticed that 'lick' appears twice, haha. Probably not a problem.

**Review by ishasinha1 — APPROVED** [review]
Just noticed a few things and none of them are blockers. Thank you, @maxkadel!

File: README.md [hunk 3]
```diff
@@ -31,3 +31,68 @@ There are capistrano tasks to connect to the Solr UI for managing solr that can
...
+scp solr_test_plan.jmx deploy@loadtest.lib.princeton.edu:~/
+```
+* Copy the keywords file onto the remote host
+```bash
+scp keywords.csv deploy@loadtest.lib.princeton.edu:~/
+```
+* SSH onto the box
+```bash
+ssh deploy@loadtest.lib.princeton.edu
+```
```
Comment (sandbergja) [inline README.md:47]: Should we delete/move the existing `test_results` directory before running?  Or is it desired to dump all results into the same directory?

**Review by sandbergja — COMMENTED** [review]
Thanks, @maxkadel !  Just one question I had while walking through these steps.

File: README.md (hunk 1 above)
Comment (maxkadel) [reply README.md:63]: Ah, thanks, I'll put it in a code block.

File: keywords.csv (hunk 2 above)
Comment (maxkadel) [reply keywords.csv]: replaced one of them with "protozoa" just cuz.

File: README.md (hunk 3 above)
Comment (maxkadel) [reply README.md:47]: Right now the box is configured so it *should* delete old results on each new test run. That's what [this config block in Princeton Ansible](https://github.com/pulibrary/princeton_ansible/blob/main/roles/loadtest/tasks/main.yml#L24-L28) should do. 

What do you think? I wasn't thinking of the server as a place for long-term storage, but that each time we run, we should save the artifacts we want to keep in other locations, but if it's easier to keep each "official" run as its own set of artifacts on the server that's ok with me, as long as we don't absolutely trust to them being there long term.

Comment (sandbergja) [reply README.md:47]: +1 to not using the server for long-term storage of test artifacts.  The only thing I was wondering about was creating the HTML report at the end -- does it have the ability to make graphs across multiple runs, and if so, how do we set it up to do so?

**Review by christinach — APPROVED** [review]

**Review by christinach — APPROVED** [review]

File: keywords.csv (hunk 2 above)
Comment (christinach) [reply keywords.csv]: @sandbergja 'protozoa' is a greek word. :)

File: README.md [hunk 4]
```diff
@@ -31,3 +31,78 @@ There are capistrano tasks to connect to the Solr UI for managing solr that can
 
 * Would it be better to go from a server on Princeton's network (e.g., the bibdata-staging worker machine), so that Developer's internet connection doesn't affect benchmarking
```
Comment (christinach) [inline README.md:32]: @maxkadel even though this is not in your PR can you edit so that it is a statement and not a question. 'It would be better...'

Comment (maxkadel) [reply README.md:32]: Good catch. I'm tempted to take out that whole section, since we now have the "loadtest" server - what do you think @christinach?

**Review by sandbergja — APPROVED** [review]
🦄

**Pull request merged**